tests/test_generated_code.py

```diff
@@ -29,6 +29,22 @@ def test_get_auth_token_without_env(model_data_with_cleanup):
     )
 
 
+def test_set_auth_token():
```

Comment:
Confirmed that, without the edit made tot he jinja template, this test will fail.

Aside: test_get_auth_token_without_env() doesn't use model_data_with_cleanup. Would you prefer the convention of passing fixtures in even if they aren't used? Or should we go ahead and remove that argument as well

Reply:
I don't have a strong opinion on this either way - But leaning on the side of not having fixtures if they aren't used.